secret-provider/src/main/scala/io/lenses/connect/secrets/config/VaultProviderConfig.scala

```diff
@@ -47,6 +47,8 @@ object VaultProviderConfig {
   val KUBERNETES_TOKEN_PATH: String = "kubernetes.token.path"
   val KUBERNETES_DEFAULT_TOKEN_PATH: String =
     "/var/run/secrets/kubernetes.io/serviceaccount/token"
+  val KUBERNETES_AUTH_PATH: String = "k8s.auth.path"
```

Comment:
can we standardise these new properties as `kubernetes` instead of `k8s` like the ones above?  Consistency is important here

Reply:
Thanks for the comment. I have changed it to maintain consistency.